dynamodb/schema.go

```diff
@@ -325,3 +326,30 @@ func numericParsable(n string) bool {
 
 	return true
 }
+
+// SetRowStats populates conv with the number of rows in each table. In
+// DynamoDB, we use describe_table api to get the number of total rows, but this
+// number is updated approximately every six hours. Recent changes might not be
+// reflected in this value.
```

Comment:
Yup, understood. This is a tradeoff here in terms of having an efficient estimate for rows vs having a more exact/up-to-data number (which would never be exact since the number of rows potentially keeps changing).

I might be worth making the implications of this clearer, so if someone sees something odd in the behaviour of harbourbridge/dynamodb, they can track it to this point.  How about editing the comment along the following lines:

// SetRowStats populates conv with the number of rows in each table. In
// DynamoDB, we use describe_table api to get the number of total rows, but this
// number is updated approximately every six hours. This means that our row
// count could be out of date by up to six hours. One of the primary uses of
// row count is for calculating progress during data conversion. As a result, if
// there have been huge changes in the number of rows in a table over the last
// six hours, the progress calculation could be inaccurate.